website/pages/en/publishing/publishing-a-subgraph.mdx

```diff
@@ -39,3 +39,24 @@ Alternatively, you can add GRT signal to a published subgraph from Graph Explore
 ## Updating metadata for a published subgraph
 
 Once your subgraph has been published to the decentralized network, you can modify the metadata at any time by making the update in Subgraph Studio dashboard of the subgraph. After saving the changes and publishing your updates to the network, they will be reflected in Graph Explorer. This won’t create a new version, as your deployment hasn’t changed.
+
+## Publishing from the CLI
+
+As of version 0.73.0, you can also publish your subgraph with the `graph-cli`.
+
+Simply type `graph publish` after doing `graph build && graph codegen` and a window will open up allowing you to connect your wallet, add metadata, and deploy your finalized subgraph to a network of your choice.
```

Comment:
```suggestion
Simply type `graph publish` after doing `graph build && graph codegen` and a window will open, allowing you to connect your wallet, add metadata, and deploy your finalized subgraph to a network of your choice.
```

Reply:
You can also switch the logical order: 

```suggestion
After running `graph build && graph codegen`, type `graph publish` , and a window will open, allowing you to connect your wallet, add metadata, and deploy your finalized subgraph to a network of your choice.
```

The only issue is that you'd have to incorrectly place commas outside to avoid messing up the command.